# exceptions_practice.py
Following class peer tutorial, an attempt at utilizing exceptions is made while implicating my own project: math module.
